modules/ROOT/pages/syntax/expressions.adoc

```diff
@@ -1246,6 +1246,8 @@ A type predicate expression can be used to verify the type of a variable, litera
 
 Where `<expr>` is any Cypher expression and `<TYPE>` is one of the following types:
```

Comment:
If these are all the existing types, could this list not just go away/be combined with the sentence below? I think it's point was to tell which of the types given in `xref::values-and-types/property-structural-constructed.adoc#types-synonyms[types and their synonyms]` could be used but if all of them are now allowed that isn't really that different than just saying something along the lines of 'all the types an their synonyms are supported'

Reply:
Or am I remembering things wrong @Lojjs?